src/lib/output/themes/default/templates/reflection.tsx

```diff
@@ -18,15 +18,13 @@ export function reflectionTemplate(context: DefaultThemeRenderContext, props: Pa
 
             {hasTypeParameters(props.model) && (
                 <section class="tsd-panel tsd-type-parameters">
```

Comment:
Any reason the `<section>` (and maybe even the condition) didn't move into the helpers? Seems like you'd still have elements you don't want if modifying these helpers.